### DISCUSSION TITLE
If OIDC is setup for subdomain, do I still need to add it in the config to be protected before auth?

### DISCUSSION THREAD

**Discussion by C8opmBM** [authelia/authelia#5008] [Q&A] [answered]
Hello,

For services that are not supported with OIDC, I have them protected like this in authelia

```
    - domain: ['abc.domain.net', 'efg.domain.net']
      policy: two_factor
```

and like this in caddy v2

```
abc.{$DOMAIN} {
        import restricted-access #this is a snippet which forwards auth to authelia
        reverse_proxy abc:port
}
```

I have also a few services setup with oidc (portainer, outline, immich, owncloud ocis, grafana, etc).
Do I still need to add these domains in the authelia config (along with the oidc section) and to the caddy `restricted-access` or this is not needed as authentication is already done via oidc?
Will this actually translate in an unneeded double authentication?
Thanks.

**Comment by james-d-elliott** [comment, marked as answer]
We recommend that the forward auth section be omitted from these subdomains generally.

**Comment by C8opmBM** [comment]
Thanks, therefore I will not include them in the authelia `default_policy` (keep only the oidc client ids setup) and will also remove the import `restricted-access` from caddy.
